tests/conftest.py

```diff
@@ -226,6 +227,58 @@ def _write_token(user):
     return _write_token
 
 
+@pytest.fixture()
+def bearer_auth_header_for_user(app, db, es, json_headers):
```

Comment:
dont repeat the same function twice ;) in one with json headers call the other one, and return result + json headers